src/app/code/community/Shopgate/Cloudapi/Model/Api2/Carts/Utility.php

```diff
@@ -148,4 +153,26 @@ private function loadInactiveCustomerQuote()
 
         return $quote;
     }
+
+    /**
+     * Creates a new quote for customerId and sets it to active
+     *
+     * @param int $customerId
```

Comment:
if you end up reusing this docblock, pretty sure it will be a string as GET params are not passed as integers via mage methods